localization/english/culture/wc_cultural_innovations_l_english.yml

```diff
@@ -0,0 +1,44 @@
+﻿l_english:
+ epidemic_conversion_bonus:0 ""Promote Culture" [councillor_task|E] Speed: #P +50%#!"
+
+ ##############
+ # Tribal Era #
+ ##############
+
+ innovation_amphibious:0 "Amphibious"
+ innovation_amphibious_desc:0 "We surface on land only when we please, but being natives to the water we can travel the seas at ease."
+
+ innovation_sea_faring:0 "Sea Faring"
+ innovation_sea_faring_desc:0 "Living on the coasts gave us a natural advantage in not only taming, but conquering the seas and those who dare to challenge us."
+
+ innovation_lorewalking:0 "Lorewalking"
+ innovation_lorewalking_desc:0 "Our ability to proficiently record history help us find ourselves through the lore of our ancestors."
+
+ innovation_hive_breeding:0 "Hive Breeding"
+ innovation_hive_breeding_desc:0 "Our massive hives allow us to breed countless workers and soldiers, always giving us the numerical advantage."
+ 
+ ######################
+ # Early Medieval Era #
+ ###################### 
+
+ innovation_time_is_money:0 "Time Is Money"
+ innovation_time_is_money_desc:0 "The flow of money is like the flow of time, it must be cherished, because without money our time here is meaningless."
+
+ innovation_elite_ranks:0 "Elite Ranks"
+ innovation_elite_ranks_desc:0 "Our ancestors did not survive through sheer will alone, we will always need the strongest of us defending those who can't defend themselves."
+
+ innovation_epidemic:0 "Epidemic"
+ innovation_epidemic_desc:0 "The plague spreads like a wildfire taking countless lives and turning them into the undead of of ranks."
+ 
+ innovation_architecture:0 "Cities Of Gold"
+ innovation_architecture_desc:0 "Our minds build cities and our hands polish them, our grand cities are well thought out and magnificent."
+ 
+ #####################
+ # High Medieval Era #
+ #####################
+
+ innovation_tinkers:0 "Tinkers"
+ innovation_tinkers_desc:0 "Natural circumstances have forced us to rely more on our brains than fists, for whats the point of having a fist with no brain to control it."
+ 
+ innovation_iron_star:0 "Iron Star"
+ innovation_iron_star_desc:0 "The foundation of all Iron Horde technology is the iron star, allowing for the construction of massive weapons of war that can wipe out whole legions with their power."
```

Comment:
The Iron Horde is from the alternate universe.

Reply:
Would better fit without the alternate universe shenanigans: 
The Iron Star is an innovative siege weapon technology introduced to the orcs through exposure to the goblin craftsmanship, allowing for the construction of massive weapons of war that can wipe out whole legions with their power.

Reply:
This one mentions the goblins. Not good.